src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/veteran-marriage-history/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const SpouseItemHeader = formData => (
+  <legend
+    className="schemaform-block-title schemaform-block-subtitle"
+    id={formData.id}
+  >
+    {formData.id.includes('0') ? 'Your former spouse(s)' : 'Next former spouse'}
```

Comment:
I didn't see where this `id` is defined, so I don't know what values are in it. What happens if the id is `02` or `10`?

Reply:
Hmm..you bring up a good point. If someone has a tenth marriage they want to report then this does go back to `Your former spouse(s)`...let me see if I can do some more digging.

Reply:
I didn't spend too much time looking, but I think id eventually ends up originating [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/fields/ArrayField.jsx#L245).  On line 263 of that file a function called `toIdSchema` is invoked which comes from the va-react-json-schema utils...which is when I stopped looking 😅 . 

I know `id` gets built off of array index, so `02` is unlikely. I think switching the check to `_0_` will fix the issue of having `10`.